### ISSUE TITLE
croc is actually good

### ISSUE THREAD

**Issue by CheaterTim** [Gwiilo/CrocodilesSuck#3]
# croc is actually good
## why?
- almost **everyone** loves the croc. so this is a big big ratio.
- also just look at the croc its soooo lovely 😻
- staff members (including senior) love the croc. and are superior to you.
- there are many types of croc. twitter croc, racing car, sam croc and many more, its impossible to hate all of them.
![croc](https://user-images.githubusercontent.com/85891155/155711496-6311e367-5f10-4a8d-b157-eeb1646b35b6.png)

## what now?
- delete this repo or change it to [croc-best](https://github.com/CheaterTim/croc-best/blob/main/croc.html)
- donate the domain to the croc foundation
- stop hating on croc
## TL;DR
don't hate on croc or attack it with a fricking hammer because of your invalid opinions.

**Comment by Gwiilo** [comment]
blasphemy

**Comment by GeorgeRNG** [comment]
when gweeloh learns of Labels 😔
